crates/biome_service/src/diagnostics.rs

```diff
@@ -192,6 +208,17 @@ pub struct ReportNotSerializable {
 )]
 pub struct NotFound;
 
+#[derive(Debug, Serialize, Deserialize, Diagnostic)]
+#[diagnostic(
+    category = "internalError/panic",
+    message = "A workspace task panicked.",
+    tags(INTERNAL)
+)]
+pub struct Panic {
+    #[description]
+    description: String,
+}
```

Comment:
I'm not sure I understand the need for this diagnostic. Ther's already a hook to catch panic errors 🤔

Reply:
The diagnostic is used _by_ the panic handler. Btw, I've updated it so that this diagnostic at least is no longer a variant of `WorkspaceError`.